bin/ruby_version_test

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Normally I've seen this setup [as part of the CI process](https://jtway.co/run-tests-against-multiple-ruby-versions-on-circleci-60e1e523e245) so that every build validates the gem against those different versions.

Reply:
going to leave that as a todo, for now